packages/lib/server/getServerErrorFromUnknown.ts

```diff
@@ -36,6 +36,11 @@ function parseZodErrorIssues(issues: ZodIssue[]): string {
 }
 
 export function getServerErrorFromUnknown(cause: unknown): HttpError {
+  const parsedStripeError = stripeInvalidRequestErrorSchema.safeParse(cause);
+  if (parsedStripeError.success) {
+    return getHttpError({ statusCode: 400, cause: parsedStripeError.data });
+  }
```

Comment:
@hbjORbj let's put this at the same lines as before. We don't want to parse ever single error unless the most popular ones fail first.

Reply:
gottcha